app/assets/stylesheets/components/_alert.scss

```diff
@@ -1,15 +1,5 @@
 @use 'uswds-core' as *;
 
-.usa-alert--info-important {
-  @include u-bg('accent-cool-darker');
-  color: color('white');
-  font-weight: bold;
-
-  &::before {
-    background-image: url('/alert/info-white.svg');
```

Comment:
Should be able to remove this file as well.

https://github.com/18F/identity-idp/blob/main/app/assets/images/alert/info-white.svg